ragna/assistants/_api.py

```diff
@@ -28,7 +28,7 @@ def __init__(
 
         self._client = httpx.Client(
             headers={"User-Agent": f"{ragna.__version__}/{self}"},
-            timeout=10,
+            timeout=None,
```

Comment:
Ok, to increase, but we should not wait indefinitely.

```suggestion
            timeout=30,
```